### PR TITLE
Link updates to release docs

### PR DESCRIPTION
**Pull request by amyblais**

### CODE REVIEW

File: operations/research-and-development/product/release-process/release-tips.md [hunk 1]
```diff
@@ -46,9 +46,4 @@
 **Release Marketing**
 
  - Blog post:
-    - Follow [guidelines](https://docs.mattermost.com/process/marketing-guidelines.html#guidelines-for-release-announcements).
-    - Audience: end-users, admin/tech.
```
Comment (jasonblais) [inline operations/research-and-development/product/release-process/release-tips.md:50]: Did you intentionally remove the audience and other lines?

**Review by jasonblais — COMMENTED** [review]
Thank you Amy! I'd be so lost without you..just one question below :)

File: operations/research-and-development/product/release-process/release-tips.md (hunk 1 above)
Comment (amyblais) [reply operations/research-and-development/product/release-process/release-tips.md:50]: Yes.

Comment (jasonblais) [reply operations/research-and-development/product/release-process/release-tips.md:50]: Makes sense, we have broader guidelines for it now

**Review by jasonblais — APPROVED** [review]

**Pull request merged**